CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to Tart
+
+Table of Contents
+-----------------
+
+- [How to Build](#how-to-build)
+- [How to Create an Issue/Enhancement](#how-to-create-an-issueenhancement)
+- [Style Guidelines](#style-guidelines)
+- [Pull Requests](#Pull-Requests)
+
+## How to Build
+
+1. Fork the repository to your own GitHub account
+2. Clone the forked repository to your local machine
+3. If using Xcode, use from Xcode 15 beta 2 (Swift 5.9)
+4. In tart/scripts/run-signed.sh, find this line:
+```bash
+codesign --sign - --entitlements Resources/tart-dev.entitlements --force .build/debug/tart
+```
+5. Replace this line with your path values in the template literals: 
+ ```bash
+ codesign --sign - --entitlements ${TART_WORKSPACE_PATH}/Resources/tart-dev.entitlements --force ${BUILT_PRODUCTS_DIR}/tart"
+```
+6. In the same file, find this line:
+```bash
+.build/debug/tart "$@"
+```
+7. Replace with your build directory
+```bash
+ ${BUILT_PRODUCTS_DIR}/tart "$@"
+```
+8. Run the script to build and run tart
+```bash
+# usage: ./scripts/run-signed.sh run ventura-base
+```
```

Comment:
This should all work if you run `./scripts/run-signed.sh list` from the root of your repository. Do you think it just easier to mention where to run the `run-signed.sh ` from? Instead of modifying it.